### PR TITLE
:rocket:  Controllers

### PR DESCRIPTION
**Pull request by SimonGideon**
# Description
In this project, I create all controllers and views for my Blog app. I add simple placeholders for your views with plain HTML.
# Related Issues
None
# Checklist

- [x] Use controllers to handle requests and render empty views.
- [x] Understand Rails naming conventions.
- [x] Params from browser requests in a safe way.

# Additional Notes

Tkanks for you review :slightly_smiling_face:

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,5 +2,10 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  get '/', to: 'users#index'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:id/post', to: 'posts#index'
+  get '/users/:id/post/:id', to: 'posts#show'
```
Comment (anagudelogu) [inline config/routes.rb:9]: - [ ] **[OPTIONAL]** Although this root structure works, it feels very repetitive and in cases where you need a lot more routes it gets hard to maintain, instead, I suggest you use the `resources` helper provided by Rails, ending up with something like this: 🤖 

```rb
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

root "users#index"

```

Please consider refactoring this! 🤓 🔧

File: spec/requests/posts_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# require 'rails_helper'
+
+# RSpec.describe 'Posts', type: :request do
+# end
```
Comment (anagudelogu) [inline spec/requests/posts_request_spec.rb:4]: - [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable, If you're not using it is okay to delete it. 🤓 🔧

File: spec/requests/users_request_spec.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# require 'rails_helper'
+
+# RSpec.describe 'Users', type: :request do
+# end
```
Comment (anagudelogu) [inline spec/requests/users_request_spec.rb:4]: - [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable, If you're not using it is okay to delete it. 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @SimonGideon, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_request_spec.rb (hunk 2 above)
Comment (SimonGideon) [reply spec/requests/posts_request_spec.rb:4]: @anagudelogu Thanks for this as part of the Ruby nest practices. The commented code are test blocks which are empty, I may need them latter but now they keep my linters falling so I had to comment them out. :slightly_smiling_face:

File: spec/requests/users_request_spec.rb (hunk 3 above)
Comment (SimonGideon) [reply spec/requests/users_request_spec.rb:4]: @anagudelogu Noted :slightly_smiling_face:

File: config/routes.rb (hunk 1 above)
Comment (SimonGideon) [reply config/routes.rb:9]: @anagudelogu Thanks for nice suggestion to make my code look better. :slightly_smiling_face:

**Review by andersonlebon — APPROVED** [review]
## Hi @SimonGideon 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimonGideon** [comment]
@andersonlebon 
messi-owned

**Pull request merged**